### DISCUSSION TITLE
Adding Dashboard testers

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#2588] [Team Posts]
Dear All,
I want to add a dashboard tester for the FPGA-OpenCL interface (https://github.com/cp2k/cp2k/pull/373). I already have a working singularity container including the Intel OpenCL SDK and Intel OpenCL FPGA-Emulation SDK. That means that the FFT-kernels would be emulated. I am planning to run these tests on our local cluster Noctua in Paderborn. (Noctua also has Stratix 10 FPGAs, so that once we have some kernels suitable for Stratix 10 we could even test them on hardware.)
In the dashboard.conf are report urls. I assume that the dashboard pull the information from there via http/https. 
Now the actual questions: 
- Are there restrictions of where theses urls can point to?
- Most of them point to http://www.cp2k.org/static/regtest/trunk. Would it be possible to upload the results from out cluster there?
Many Thanks

**Comment by dev-zero** [comment]
A lot of regtesters are actually uploading the reports via SSH instead. @oschuett can setup a user for that on `sham` (name of the host VM behind dashboard.cp2k.org)

**Comment by robertschade** [comment]
That is what I thought, but it is not documented anywhere. It would be great if I could also do it this way because then I don't have to set up or find an intermediate webserver. 
@oschuett should I send you an ssh-pubkey?

**Comment by oschuett** [comment]
Yes, while any public url would do, you're welcome to use the cp2k.org server. Just send me your pubkey at ole.schuett@cp2k.org.
